infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/TestDataSender.java

```diff
@@ -34,7 +36,18 @@ public TestDataSender() {
         }
 
         CompletableFuture<LogReplicationEntry> cf = new CompletableFuture<>();
-        LogReplicationEntry ack = LogReplicationEntry.generateAck(message.getMetadata());
```

Comment:
nit - probably we should rename this to setter of something.. because it is only setting the metadata.

Reply:
Yes, I had this comment on another PR and that PR is addressing it, so I'm not going to touch this here to not conflict.